tests/testthat/test_utils.R

```diff
@@ -91,6 +106,78 @@ test_that("test_writeTables", {
+                                             `19045` = structure(list(reference_feature_id = structure(c(6.28056556040491e-257,
+                                                                                                         4.37872094299259e+26, 4.97487068543727e-50, 1.96129357589358e+225,
+                                                                                                         6.28056556040491e-257, 4.37872094299259e+26, 4.97487068543727e-50,
+                                                                                                         2.22889297151144e-67, 1.96129357589358e+225, 0, 0, 0, 0,
+                                                                                                         0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0), class = "integer64"),
+                                                                      experiment_feature_id = structure(c(1.39189055078857e+239,
+                                                                                                          1.64937896938207e+218, 1.3591112627868e-271, 3.36180583288588e-229,
+                                                                                                          2.16310643042558e-237, 5.82423961711318e+254, 9.33806679439292e+229,
+                                                                                                          1.58730393504064e-226, 7.21410639770476e+159, 0, 0, 0,
+                                                                                                          0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0), class = "integer64")), row.names = c(NA, -30L), class = c("data.table", "data.frame")),
+                                             `19800` = structure(list(reference_feature_id = structure(c(4.72799309850901e-140,
+                                                                                                         1.98100991301857e-234, 4.7297745210988e+87, 4.72799309850901e-140,
+                                                                                                         4.7297745210988e+87, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0), class = "integer64"),
+                                                                      experiment_feature_id = structure(c(8.56560434196891e+243,
+                                                                                                          1.62593959656114e+49, 4.1149805476014e-97, 1.94083176316202e-40,
+                                                                                                          1.29045098420263e-28, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0), class = "integer64")), row.names = c(NA, -15L), class = c("data.table", "data.frame")))
+            # Write out alignment map to disk
+            writeOutFeatureAlignmentMap(multiFeatureAlignmentMap, oswMerged=TRUE, fileInfo)
```

Comment:
Do you delete the file after testing? or is it modifying the input file?

Reply:
I modify the input file by adding a table. I deleted the table after checking, in line 176.

Reply:
The inst/extdata/osw/merged.osw file changes everytime I run tests. It shows up during `git status`

Reply:
That's because the file gets modified. The input file being the merged.osw file gets modified by adding a table, and then deleting the table in line 176.

Reply:
Is there a way to revert it to previous state so that it doesn't show up in git status? Like copying the file from R and deleting it after test.

Reply:
If any other function updates the file, how would anyone know that file is modified?

Reply:
I updated the test to make a copy of the test OSW file, modify the copied test file, and then remove the copied OSW file after testing. I also added a test for just writing to disk a TSV file, and then remove the tsv file after testing. If you want to avoid the copying of the OSW file, then you can just remove the writing to OSW part, and just test for writing to TSV.